compose/release-notes.md

```diff
@@ -7,6 +7,90 @@ redirect_from:
   - /release-notes/docker-compose/
 ---
 
+# 2.5.0
+(2022-04-29)
+
+### Bug Fixes
+- Fixing panic with `compose down` command when -p flag specified. Closes [#9353](https://github.com/docker/compose/issues/9353){:target="_blank" rel="noopener" class="_"}.
```

Comment:
@dockertopia Could you change these to the past tense form? Also, could you update `Closes` to 'Fixes' when we reference external issues?